extensions/ql-vscode/.storybook/vscode-theme-addon/manager.tsx

```diff
@@ -1,13 +1,14 @@
 import * as React from "react";
-import { addons, types } from "@storybook/manager-api";
+import { addons } from "@storybook/manager-api";
+import { Addon_TypesEnum } from "@storybook/types";
 import { ThemeSelector } from "./ThemeSelector";
 
 const ADDON_ID = "vscode-theme-addon";
 
 addons.register(ADDON_ID, () => {
   addons.add(ADDON_ID, {
     title: "VSCode Themes",
-    type: types.TOOL,
+    type: Addon_TypesEnum.TOOL,
```

Comment:
There is no clear guidance how to replace `types`, however `Addon_TypesEnum` from storybook/types is used in the source for `types`, so I think this should work.

Reply:
I also can't see any documentation on intended migration strategy. I tracked down https://github.com/storybookjs/storybook/pull/24834 where this change was made and there's nothing there either.

I agree I think using `Addon_TypesEnum` seems fine.